apps/privacy-panel/js/app_list.js

```diff
@@ -102,18 +102,18 @@ define([], function() {
    *   .origin:      application origin
    */
   function _makeAppRepresentation(app) {
-    var manifest = app.manifest || app.updateManifest || {};
-
+    var manifest = new ManifestHelper(app.manifest ?
+        app.manifest : app.updateManifest);
+    
     var trust = 'web';
     if (manifest.type === 'certified' || manifest.type === 'privileged') {
       trust = manifest.type;
     }
-
-    var name = manifest.name;
+    var name = manifest.displayName;
     if (manifest.locales &&
         manifest.locales[_lang] &&
-        manifest.locales[_lang].name) {
-      name = manifest.locales[_lang].name;
+        manifest.locales[_lang].displayName) {
```

Comment:
You dont need this here - the ManifestHelper does the locale/lang resolution for you. You can simple access .displayName